Jenkinsfile

```diff
@@ -43,11 +43,14 @@ try {
       }
 
       stage ("Configure npm") {
-        withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'npm-moltin-moltinbot-password', usernameVariable: 'NPM_USERNAME', passwordVariable: 'NPM_PASSWORD']]) {
-          sh "docker run -e NPM_USER=$NPM_USERNAME -e NPM_PASS=\"$NPM_PASSWORD\" -e NPM_EMAIL=$NPM_EMAIL bravissimolabs/generate-npm-authtoken > .npmrc.tmp"
+        withCredentials([
+          string(credentialsId: 'npm-moltin-moltinbot-email', variable: 'NPM_EMAIL'),
+          usernamePassword(credentialsId: 'npm-moltin-moltinbot-password', usernameVariable: 'NPM_USERNAME', passwordVariable: 'NPM_PASSWORD')
+        ]) {
+          sh "docker run -e NPM_USER=$NPM_USERNAME -e NPM_PASS=\"$NPM_PASSWORD\" -e NPM_EMAIL=$NPM_EMAIL bravissimolabs/generate-npm-authtoken > .npmrc"
```

Comment:
We need to keep this with the `.npmrc.tmp` for this project as it will be replace when using the `sematic-release` tool